tools/README.md

```diff
@@ -91,15 +91,15 @@ the following shows this process (building and pushing prebuilt images) for
 
 ```shell
 bin/prepare_prebuilt_workers \
-     -l cxx:master \
-     -l go:master \
+     -l cxx:grpc/grpc:master \
```

Comment:
I don't think it is good to make a change that is not backward-compatible.

I would keep the instructions as they are now and add a section about running from a forked repo. In that section I would say that you can build from a forked repo by adding the repo in addition to the GITREF.

Reply:
I have updated it to make if backward-compatible, meaning it takes both cxx:master  and cxx:grpc/grpc:master, would that work? Thank you